src/library/atmosphere/simple_air_density_model.hpp

```diff
@@ -0,0 +1,20 @@
+/**
+ * @file simple_air_density_model.hpp
+ * @brief Standard earth's atmospheric density model
+ */
+#ifndef S2E_LIBRARY_SIMPLE_AIR_DENSITY_MODEL_HPP_
+#define S2E_LIBRARY_SIMPLE_AIR_DENSITY_MODEL_HPP_
+
+namespace libra::atmosphere {
+
+/**
+ * @fn CalcAirDensityWithStandardModel
```

Comment:
[NITS] 関数名の修正をお願いします．
```suggestion
 * @fn CalcAirDensityWithSimpleModel
```

Reply:
done